test/Vegetation/plant_hydraulics_test.jl

```diff
@@ -12,6 +12,58 @@ using Insolation
 using Dates
 
 include(joinpath(pkgdir(ClimaLSM), "parameters", "create_parameters.jl"))
+@testset "LAI assertions" begin
```

Comment:
Nice job covering all the cases :)

Reply:
:D thanks!

Ill add in the checks for the ones we just added.